resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/TestConstants.scala

```diff
@@ -34,4 +34,6 @@ object TestConstants {
   val CONFIG_KEY_IMAGE_TAG_FILE = "spark.kubernetes.test.imageTagFile"
   val CONFIG_KEY_IMAGE_REPO = "spark.kubernetes.test.imageRepo"
   val CONFIG_KEY_UNPACK_DIR = "spark.kubernetes.test.unpackSparkDir"
+  val CONFIG_DRIVER_REQUEST_CPU = "spark.kubernetes.test.driver.cpu"
+  val CONFIG_EXECUTOR_REQUEST_CPU = "spark.kubernetes.test.executor.cpu"
```

Comment:
ditto.

Reply:
rename to `spark.kubernetes.test.executorRequestCores`